src/js/actions/export.js

```diff
@@ -163,11 +166,44 @@ define(function (require, exports) {
                     filePath: "",
                     status: ExportAsset.STATUS.ERROR
                 };
-                return this.transfer(updateExportAsset, document, layer, assetIndex, assetProps);
+                return this.transfer(updateExportAsset, document, _layers, assetIndex, assetProps);
             }) ;
     };
 
     /**
+     * Insert an asset (or assets) into a document or set of layers, by inserting into the existing list
+     * at the given index.
+     * Sync to PS metadata afterwards
+     *
+     * 
+     * @private
+     * @param {Document} document
+     * @param {Immutable.Iterable.<Layer>=} layers if not supplied, then doc-level assets
+     * @param {number} assetIndex index of this asset within the layer's list to append props 
+     * @param {object|Array.<object>} props ExportAsset-like properties to be merged, or an array thereof
+     * @return {Promise}
+     */
+    var _insertAssetsAtIndex = function (document, layers, assetIndex, props) {
+        var documentID = document.id,
+            layerIDs = layers && collection.pluck(layers, "id") || undefined,
+            assetPropsArray = Array.isArray(props) ? props : [props],
+            payload;
+
+        payload = {
```

Comment:
I think the preferred style is just to declare payload on this line instead of declaring it and then defining it.